src/main/java/com/google/javascript/gents/TypeConversionPass.java

```diff
@@ -272,43 +287,18 @@ public void visit(NodeTraversal t, Node n, Node parent) {
     }
   }
 
-  /**
-   * Go through all JSDocs and rename everything from typesToRename map. For example: when a class
-   * inner typedef is converted to a top level interface, all the references of the typedef in any
-   * JSDocs are renamed to the top level interface name.
-   */
-  private class TypeAliasConverter extends AbstractPostOrderCallback {
-    @Override
-    public void visit(NodeTraversal t, Node n, Node parent) {
-      // Go through JSDocs, find and replace typedefs
-      @Nullable JSDocInfo bestJSDoc = NodeUtil.getBestJSDocInfo(n);
-      if (bestJSDoc != null) {
-        JSTypeExpression type = bestJSDoc.getType();
-        if (type != null) {
-          maybeRename(type.getRoot());
-        }
-
-        for (String parameterName : bestJSDoc.getParameterNames()) {
-          @Nullable JSTypeExpression parameterType = bestJSDoc.getParameterType(parameterName);
-          if (parameterType != null) {
-            maybeRename(parameterType.getRoot());
-          }
-        }
+  private void convertTypeAlias() {
+    Map<String, FileModule> fileMap = modulePrePass.getFileMap();
+    for (String typeName : typesToRename.keySet()) {
```

Comment:
prefer `entrySet()`

`for (Entry<String, FileModule> entry : typesToRename.entrySet())`